services/business/service/tenant-service.go

```diff
@@ -101,21 +121,27 @@ func (service *TenantService) ReadTenant(
 
 // UpdateTenant update an existing tenant
 // context: Mandatory The reference to the context
-// request: Mandatory. The request to update an esiting tenant
+// request: Mandatory. The request to update an exiting tenant
```

Comment:
Now that you fixed the typo, make sure it is written as "existing", not exiting ;-)